Mathlib.lean

```diff
@@ -1124,6 +1124,7 @@ import Mathlib.AlgebraicGeometry.EllipticCurve.Jacobian.Basic
 import Mathlib.AlgebraicGeometry.EllipticCurve.Jacobian.Formula
 import Mathlib.AlgebraicGeometry.EllipticCurve.Jacobian.Point
 import Mathlib.AlgebraicGeometry.EllipticCurve.ModelsWithJ
+import Mathlib.AlgebraicGeometry.EllipticCurve.Modular.TateNormalForm
```

Comment:
Maybe have a `NormalForm` folder and call this `NormalForms.Tate`? What else do you intend to be in `Modular`?